stock_release_channel/models/stock_release_channel.py

```diff
@@ -564,7 +564,11 @@ def get_action_picking_form(self):
 
     @staticmethod
     def _pickings_sort_key(picking):
-        return (-int(picking.priority or 1), picking.date_priority, picking.id)
+        return (
+            -int(picking.priority or 1),
+            picking.date_priority or picking.scheduled_date,
```

Comment:
I prefer this then
```suggestion
            picking.date_priority or picking.create_date,
```

Reply:
Because both date_priority and create_date default to now() while scheduled_date is not
Nevertheless, date_priority should always have a value

Reply:
Yes they are both default to now, but also the scheduled_date. 
Because it gets computed from the date field of the moves. 
https://github.com/odoo/odoo/blob/e8834cb00b82e0bee7f6338aef940037a3e49471/addons/stock/models/stock_picking.py#LL291C8-L291C8
https://github.com/odoo/odoo/blob/e8834cb00b82e0bee7f6338aef940037a3e49471/addons/stock/models/stock_picking.py#L521
https://github.com/odoo/odoo/blob/e8834cb00b82e0bee7f6338aef940037a3e49471/addons/stock/models/stock_move.py#L39

Should we not process a picking/move which is created later, as an other picking/move, but with an earlier scheduled_date/date?

Reply:
Yes this is looking good, i change it like you mentioned.